docs/site/Sequence.md

```diff
@@ -41,20 +41,17 @@ class DefaultSequence {
 In the example above, `route`, `params`, and `result` are all Elements. When building sequences, you use LoopBack Elements to respond to a request:
 
 - [`Route`](http://apidocs.loopback.io/@loopback%2frest/#Route)
-- [`Request`](http://apidocs.strongloop.com/loopback-next/)  - (TBD) missing API docs link
+- [`Request`](http://apidocs.strongloop.com/@loopback%2frest/#ParsedRequest)  - (TBD) missing API docs link
 - [`Response`](http://apidocs.strongloop.com/loopback-next/) - (TBD) missing API docs link
 - [`OperationRetVal`](http://apidocs.loopback.io/@loopback%2frest/#OperationRetval)
 - [`Params`](http://apidocs.strongloop.com/loopback-next/) - (TBD) missing API docs link
```

Comment:
Can you change this to point to `apidocs.loopback.io`. 

- Same for `Request` and `Response` above.